Wireframe/css/style.css

```diff
@@ -0,0 +1,89 @@
+*{
+text-align: center;
+color: white;
+margin: 0;
+}
+body{
+    background:rgb(71, 70, 70);
+}
+
+#title{
+    margin: 20px;}
+#subtitle{
+    margin: 20px 0px 30px 0px;
+}
+
+article{
+    border: 1px white solid;
+    text-align: left;
+}
+
+.first img , .second img{
+    object-fit: cover; 
+}
+
+
+
+ .first{
+    width: 69%;
+    margin: auto;
+    height: 600px;
+    margin-bottom:30px ;
+} 
+a:hover{
+    background: #000;
+}
+
+article.first :first-child{
+    filter:blur(2px);
+}
+.second , .third{
+    display: inline-block;
+    width: 33.5%;
+    height: 550px;
+    margin:0px 15px 100px  15px;
+    
+}
+
+
+footer{
+    border-top: solid 1px  ;
+    position: fixed;
+    width: 100%;
+    height: 60px;
+    bottom: 0;
+    background-color: #f0f0f0;
+}
+
+footer h3 {
+    color: black;
+    margin-top: 15px;
+}
+a{
+    border: 1px white solid;
+    border-bottom: none;
+    padding: 15px ;
+    margin-left:20px;
+    text-decoration: none;
+}
+body article :first-child(){
+    color: blue;
+    border: 1px blueviolet solid
+}
+
+article h1, article h3 {
+text-align: left;
+padding-left: 20px;
+}
+.first h1{
```

Comment:
You as setting all `h1`'s inside all articles with `margin: 8px 0px ;`

You can replace lines these lines
```
.first h1{
    margin: 8px 0px ;
}
.second h1,.third h1{
    margin: 8px 0px ;
}
```
With
```
article h1{
    margin: 8px 0px ;
}
```

This is simpler and easier to read and does the same thing. Saves time and code and makes future changes clearer.

Same with these lines too
```
.first h3{
    margin: 15px 0px 49px 0px ;
}
.second h3,.third h3{
    margin: 15px 0px 49px 0px ;
}
```

```
article h3{
    margin: 15px 0px 49px 0px ;
}
```

So you could simplify lines 78 to 89 to 
```
article h1{
    margin: 8px 0px ;
}

article h3{
    margin: 15px 0px 49px 0px ;
}
```